instrumentation/packaging/fpm/common.sh

```diff
@@ -24,16 +24,18 @@ PKG_DESCRIPTION="Splunk OpenTelemetry Auto Instrumentation"
 PKG_LICENSE="Apache 2.0"
 PKG_URL="https://github.com/signalfx/splunk-otel-collector"
 
-LIBSPLUNK_INSTALL_PATH="/usr/lib/splunk-instrumentation/libsplunk.so"
-JAVA_AGENT_INSTALL_PATH="/usr/lib/splunk-instrumentation/splunk-otel-javaagent.jar"
-CONFIG_INSTALL_PATH="/usr/lib/splunk-instrumentation/instrumentation.conf"
+INSTALL_DIR="/usr/lib/splunk-instrumentation"
+LIBSPLUNK_INSTALL_PATH="${INSTALL_DIR}/libsplunk.so"
+JAVA_AGENT_INSTALL_PATH="${INSTALL_DIR}/splunk-otel-javaagent.jar"
+CONFIG_INSTALL_PATH="${INSTALL_DIR}/instrumentation.conf"
+CONFIG_PATH="$REPO_DIR/instrumentation/install/instrumentation.conf"
+EXAMPLES_INSTALL_DIR="${INSTALL_DIR}/examples"
+EXAMPLES_DIR="${FPM_DIR}/examples"
 
 JAVA_AGENT_RELEASE_PATH="${FPM_DIR}/../java-agent-release.txt"
 JAVA_AGENT_RELEASE_URL="https://github.com/signalfx/splunk-otel-java/releases/"
 
-POSTINSTALL_PATH="$FPM_DIR/postinstall.sh"
```

Comment:
What is the ld.so.preload removal process to replace this?

Reply:
The removal happens in preuninstall.sh (see below), which has not been changed. So it should always be removed, even if added externally.